S/scopehal/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "scopehal"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/azonenberg/scopehal.git", "db29f07e396667135ccf02b37e27f68b8d4998a1"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/
+# git clone --recursive https://github.com/azonenberg/scopehal.git
+cd scopehal/
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+git submodule update --init
+cd ..
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN%.*}_gcc.cmake \
+    -DCMAKE_BUILD_TYPE=Release \
+    .
+make
+make install
+install_license scopehal/LICENSE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```

Comment:
> Two things to note
>
> * author considers library practically 64b only due to large memory requirements
> * author says ARM is not yet supported

So I guess we can do
```suggestion
platforms = filter!(p -> BinaryBuilder.proc_family(p) != "arm" || nbits(p) == 64, supported_platforms())
```
?

Reply:
Well, I just fixed 32b linux. Let's hold on to this for a bit and look what other errors we can easily solve.

Reply:
Musl, and all non-Intel platforms look a bit hopeless at the moment, my new suggestion is
```suggestion
platforms = filter!(p -> BinaryBuilder.proc_family(p) == "intel" && libc(p) != "musl", supported_platforms())
```